tests/test_jetpack.py

```diff
@@ -456,23 +433,12 @@ def test_outdated_sdkversion():
     """
 
     xpi = XPIManager("tests/resources/jetpack/jetpack-1.8-outdated.xpi")
-    err = _do_test(xpi, allow_old_sdk=False)
+    with accepted_version("frameworks", "jetpack", "versions", "1.8"):
+        err = _do_test(xpi, allow_old_sdk=False)
 
     assert err.failed()
     # Make sure we don't have any version mismatch warnings
+    eq_(err.errors, [])
     eq_(len(err.warnings), 1)
     eq_(err.warnings[0]["id"][2], "outdated_version")
 
-
-def test_future_sdkversion():
```

Comment:
No. The expected behavior is that we should warn when a future version of the SDK is used. But this test wasn't actually testing that, since 1.8 had not been a future version in a very long time.